ui/src/app/workflows/components/workflow-details/workflow-details.tsx

```diff
@@ -136,14 +140,20 @@ export class WorkflowDetails extends React.Component<RouteComponentProps<any>, W
                                                     selectedNodeId={this.selectedNodeId}
                                                     nodeClicked={nodeId => this.selectNode(nodeId)}
                                                 />
-                                            )) || (
-                                                <WorkflowTimeline
-                                                    workflow={this.state.workflow}
-                                                    selectedNodeId={this.selectedNodeId}
-                                                    nodeClicked={node => this.selectNode(node.id)}
-                                                    ref={timeline => (this.timelineComponent = timeline)}
-                                                />
-                                            )}
+                                            )) ||
+                                                (this.selectedTabKey && (
+                                                    <EventsPanel
```

Comment:
![image](https://user-images.githubusercontent.com/1142830/89853538-82fc7c00-db46-11ea-998d-d10ff7fce72d.png)

Reply:
we should probably have

`kind in (Workflow,PodDisruptionBudget,PersistentVolumeClaim)`

Reply:
```
[acollins8@intuitdepe1a95 argo (⎈ |k3s-default:argo) (feat-3725)]$ k get event --field-selector='involvedObject.kind in ("Pod")'
Error from server (BadRequest): Unable to find "/v1, Resource=events" that match label selector "", field selector "involvedObject.kind in (\"Pod\")": invalid selector: 'involvedObject.kind in ("Pod")'; can't understand 'involvedObject.kind in ("Pod")'

```